Source/LibationUiBase/GridView/GridEntry.cs

```diff
@@ -128,6 +132,7 @@ public void UpdateLibraryBook(LibraryBook libraryBook)
 		protected virtual DateTime GetPurchaseDate() => LibraryBook.DateAdded;
```

Comment:
Below this line, add:
```c#
protected virtual DateTime? GetIncludedUntil() => LibraryBook.IncludedUntil;
```

Then go to SeriesEntry at the end of the file: https://github.com/rmcrackan/Libation/blob/2c882e883d49c01d7428437e1f7ea51ab5df2f07/Source/LibationUiBase/GridView/SeriesEntry.cs#L104-L105


Add this line to the end of the class:

```c#
protected override DateTime? GetIncludedUntil() => Children.Count == 0 ? default : Children.Min(c => c.LibraryBook.IncludedUntil);
```